src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Data/PathParser.cs

```diff
@@ -169,18 +166,18 @@ public SourceValueInfo[] Parse(string path)
 
             if (_error == null)
             {
-                result = new SourceValueInfo[_al.Count];
-                _al.CopyTo(result);
+                result = new SourceValueInfo[_sourceValueInfos.Count];
+                _sourceValueInfos.CopyTo(result);
```

Comment:
```suggestion
                result = _sourceValueInfos.ToArray();
```

Reply:
Yeah, should be faster and clearer :) Thanks :)